dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/DatadogCoreTest.kt

```diff
@@ -728,6 +729,13 @@ internal class DatadogCoreTest {
         )
         val mockCoreFeature = mock<CoreFeature>()
         testedCore.coreFeature = mockCoreFeature
+        val mockExecutorService: FlushableExecutorService = mock()
+        whenever(mockCoreFeature.persistenceExecutorService) doReturn mockExecutorService
+        whenever(mockExecutorService.submit(any())) doAnswer {
```

Comment:
I see so this is because we've made those methods async ?

Reply:
Yep